griptape/loaders/email_loader.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+from attr import astuple, define, field
+from griptape import utils
+from griptape.artifacts import ErrorArtifact, ListArtifact, TextArtifact
+from griptape.loaders import BaseLoader
+from typing import Optional
+import imaplib
+import logging
+import mailparser
+
+
+@define(frozen=True)
+class EmailQuery:
+    """An email retrieval query"""
+
+    label: str = field(kw_only=True)
+    """Label to retrieve emails from such as 'INBOX' or 'SENT'"""
+
+    key: Optional[str] = field(default=None, kw_only=True)
+    """Optional key for filtering such as 'FROM' or 'SUBJECT'"""
+
+    search_criteria: Optional[str] = field(default=None, kw_only=True)
+    """Optional search criteria to filter emails by key"""
+
+    max_count: Optional[int] = field(default=None, kw_only=True)
+    """Optional max email count"""
```

Comment:
Can you please update the class doc string to match [Google's docstring style](https://google.github.io/styleguide/pyguide.html#384-classes)? It's what our auto generated reference docs expects.